src/scripting_api/event.js

```diff
@@ -94,6 +94,11 @@ class EventDispatcher {
           // Initialize named actions before calling formatAll to avoid any
           // errors in the case where a formatter is using one of those named
           // actions (see #15818).
+
+          // The user has decided to open this pdf, hence we enable
+          // userActivation.
+          this.userActivation();
+
           this._document.obj._initActions();
```

Comment:
Shouldn't this be moved up slightly, to not appear *in between* the pre-existing comment and code?
```suggestion
          // The user has decided to open this pdf, hence we enable
          // userActivation.
          this.userActivation();
          // Initialize named actions before calling formatAll to avoid any
          // errors in the case where a formatter is using one of those named
          // actions (see #15818).
          this._document.obj._initActions();
```